documentation/styleguide-doc.md

```diff
@@ -0,0 +1,167 @@
+# Documentation Coding Style
+
+## Where are the docs?
+
+TL;DR version: [`docs`](https://github.com/StackExchange/dnscontrol/tree/master/docs) is the [marketing website](https://dnscontrol.org). [`documentation`](https://github.com/StackExchange/dnscontrol/tree/master/documentation) is the [docs.dnscontrol.org](https://docs.dnscontrol.org/) website. (Yes, the names are backwards!)
+
+**The two websites**
+
+1. <https://dnscontrol.org/>
+   * The main website
+   * Source code: [`docs`](https://github.com/StackExchange/dnscontrol/tree/master/docs)
+   * Mostly "marketing" for the project.
+   * Rarely changes.  Updated via GitHub "pages" feature.
+2. <https://docs.dnscontrol.org/>
+   * Project documentation
+   * Source code: [`documentation`](https://github.com/StackExchange/dnscontrol/tree/master/documentation)
+   * Users and developer documentation
+   * Changes frequently.  Updated via [GitBook](https://www.gitbook.com/)
+
+**The directory structure**
+
+Within the git repo, docs are grouped:
+
+* [`documentation/`](https://github.com/StackExchange/dnscontrol/tree/master/documentation): general docs
+* [`documentation/providers/`](https://github.com/StackExchange/dnscontrol/tree/master/documentation/providers/): One file per provider
+* [`documentation/functions/`](https://github.com/StackExchange/dnscontrol/tree/master/documentation/functions/): One file per `dnsconfig.js` language feature
+* [`documentation/assets/gcloud/`](https://github.com/StackExchange/dnscontrol/tree/master/documentation/assets/gcloud): Images for page Google Cloud DNS (PNGs only, please!)
+
+## How to add a new page?
+
+1. Add the page to the `documentation` (or a sub folder)
```

Comment:
```suggestion
1. Add the page to the `documentation` folder (possibly a sub folder)
```

Reply:
_<https://docs.dnscontrol.org/~/revisions/BRtPCFXEcJg89ztpW8cK/developer-info/styleguide-doc#how-to-add-a-new-page>_